downstream/modules/platform/proc-gw-team-access-resources.adoc

```diff
@@ -13,10 +13,10 @@ Direct team access cannot be granted to {MenuACAdminRemoteRegistries} resources.
 
 .Procedure
 
-. From the navigation panel, select a resource to which you want to provide team access. For example, {MenuAETemplates}.
+. From the navigation panel, select a resource that you want to provide a team access to. For example, {MenuAETemplates}.
 . Select the *Team Access* tab.
```

Comment:
Hey @hherbly . I've been doing due diligence to help out as much as possible, including following along with procedures since doc is a pretty large undertaking. I might not be following properly here, but I'm not seeing a team access tab in our CI daily development deployments. You've worked more extensively with this. So I'll just do a high-level and just say the tabs don't seem to be visible. Adding a screen shot and admitting that I might be logging in to the wrong resource. If so, please feel free to ignore this. 

Screen shot: 
<img width="1754" height="956" alt="Screenshot From 2025-09-18 17-09-48" src="https://github.com/user-attachments/assets/b2825345-2398-483f-b656-79d59d246d86" />